Simplified/NYPLAccountSignInViewController.h

```diff
@@ -17,4 +18,6 @@
 // and pin.
 + (void)authorizeUsingExistingBarcodeAndPinWithCompletionHandler:(void (^)(void))handler;
 
++ (void)authorizeUsingIntermediaryWithCompletionHandler:(void (^)(void))handler;
```

Comment:
This was used once only. I removed it and added
`requestCredentialsUsingExistingBarcode: authorizeImmediately: completionHandler:` function to the header file, so that the method is accessible